.github/actions/generate-coverage/tests/conftest.py

```
@@ -0,0 +1,17 @@
+import pytest
+import os
+import sys
+from pathlib import Path
+
+ROOT = Path(__file__).resolve().parents[4]
+sys.path.insert(0, str(ROOT))
+
+from shellstub import StubManager
+
+@pytest.fixture
+def shell_stubs(tmp_path, monkeypatch) -> StubManager:
```

Ruff (0.11.9)

Missing type annotation for function argument `tmp_path`

(ANN001)